charts/consul/values.yaml

```diff
@@ -109,7 +109,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: "hashicorp/consul:1.13.2"
+  image: "hashicorp/consul:1.13.3"
 
   # Array of objects containing image pull secret names that will be applied to each service account.
   # This can be used to reference image pull secrets if using a custom consul or consul-k8s-control-plane Docker image.
```

Comment:
Shouldn't the `consul-k8s-control-plane:0.49.0` be bumped to `0.49.1` as well?

Reply:
i think the make prepare-release step does this

Reply:
i wasn't sure if make-prepare release goes into this commit also?

Reply:
you're definitely right that these changes go here

Reply:
i think the engineering docs are a bit confusing since they say to not merge the prepare release bit

Reply:
thanks for catching!

Reply:
Oh, okay that makes sense, I was wondering what was going on there. Yeah maybe run the `make prepare-release` step on this PR?

Reply:
Feel free to alter anything, maybe it doesn't flow as well for a patch release?

Reply:
ya i'll try updating it to follow a patch release-- i think you're totally right it may just be slightly different steps.